src/middlewared/middlewared/plugins/tunables.py

```diff
@@ -99,6 +113,24 @@ async def do_delete(self, id):
         Delete Tunable of `id`.
         """
         tunable = await self._get_instance(id)
+        await self.lower(tunable)
+        if tunable['type'].lower() == 'sysctl':
+            # Restore the default value, if it is possible.
+            value_default = None
+            try:
+                value_default = self.__default_sysctl[tunable["var"]]
```

Comment:
Lets use get_default_value so we dont mess with __default_sysctl outside the getter/setter